packages/@okta/vuepress-site/docs/reference/rl-dashboard/index.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Rate limit dashboard
+excerpt: >-
+  Understand how to use the rate limit dashboard, how to access it, how to use the multiple views using the graphs, and how to investigate a rate limit violation
+---
+
+# Rate limit dashboard
+
+The rate limit dashboard is a tool that helps you understand the usage of each endpoint and investigate potential [rate limit](/docs/reference/rate-limits/) issues. The dashboard provides:
+
+* The ability to track the endpoint's use
+* Alerts when you are about to hit or have hit the rate limit
+* The ability for you to investigate the issue when an endpoint has reached the rate limit
+
+You can find out more about the rate limit dashboard in these sections:
+
+ Section                                                                          | Description                                                                                     |
+| --------------------------------------------------------------------------------  | -------------------------------------------------------------------------               |
+| [About the rate limit dashboard](#about-the-rate-limit-dashboard)                                                          | A summary of the rate limit dashboard                     |
+| [Access the dashboard](#access-the-dashboard)                                                                  | The different types of notifications that provide access to the dashboard |
+| [Key components of the dashboard](#key-components-of-the-dashboard)                                                        | RA description of the key components on the dashboard                                                      |
```

Comment:
"RA description" is this a typo? What does RA mean?

Reply:
Fixed, thanks!